### PR TITLE
LondonClass8 - Marina Almeida - Full Stack Project - Level 100

### PR DESCRIPTION
**Pull request by MarinaAlmeida20**

### CODE REVIEW

**Comment by MarinaAlmeida20** [comment]
@jackhclee

File: client/src/App.css [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+
+.add-video-and-search {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-bottom: 4rem;
+}
+
+.add-video{
```
Comment (jackhclee) [inline client/src/App.css:13]: better to have a aligned css file syntax / formatting

File: client/src/components/Votes.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  const decrementCount = () => {
+    setCount((prevState) => prevState - 1);
+  };
+
+  return (
+    <>
+      <div className="votes">
+        <i className="fas fa-thumbs-up vote" onClick={incrementCount}></i>
```
Comment (jackhclee) [inline client/src/components/Votes.jsx:18]: consider `<i/>` for empty tag

File: client/src/components/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import "../App.css";
+
+function Main({ data }) {
+  const [wordEntered, setWordEntered] = useState("");
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [allData, setAllData] = useState(data);
+  const [showForm, setShowForm] = useState(false);
+
+  const handleSubmit = async (e) => {
```
Comment (jackhclee) [inline client/src/components/Main.jsx:17]: Great, you should also check if the random number has been used already.

File: client/src/components/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    e.preventDefault();
+
+    const newData = {
+      id: Math.floor(Math.random() * 1000),
```
Comment (jackhclee) [inline client/src/components/Main.jsx:21]: Great, you should also check if the random number has been used already.

File: client/src/components/Votes.jsx (hunk 2 above)
Comment (jackhclee) [reply client/src/components/Votes.jsx:18]: ok. How could you store the changed vote count back to the backend array?

**Pull request closed without merging**